xml/decoders/LDH_SyM-v60-port.xml

```diff
@@ -520,38 +519,9 @@
 			default="120" item="EMF Option 3" comment="Rango 1-255" tooltip="CV 64">
 			<decVal min="1" max="255"/>        	</variable>
 
-		<variable label="Speed Table" CV="67" default="1">
+		<variable item="Speed Table" CV="67" default="1">
 			<speedTableVal/>         </variable>
 
-    <variable label="Valores" CV="67" default="1"><decVal/></variable>
```

Comment:
Why are these lines removed? Are they not needed? I'm not against it, just checking.

Reply:
They're not needed. "Valores" isn't in any pane, so they're not displayed.  I think they're there to put the speed table CVs into the CV table, but that's being done automatically now.

Thanks for the careful checking!

Reply:
Also, having them there caused the schema validation to fail, as you can't have more than one variable with the same name these days.